### ISSUE TITLE
Train, Val and Test Metrics Does Not Change At All After Adding Background Images

### ISSUE THREAD

**Issue by ghost** [ultralytics/yolov5#10193]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### YOLOv5 Component

Training

### Bug

Hi, my training, validation and testing results on my custom dataset do not change at all after adding negative samples to my training dataset.

I copied the negative samples (images) to the images/train directory which contained the other training images, but see no improvement or even change in any metrics - the training loss, validation loss and test metrics (Precision, Recall, mAP) are exactly the same as before adding these negative samples. 

It does not seem like the negative samples are being used. Can someone please let me know if this is expected?

My training dataset has 1500 images and I added 150 (10%) images in addition (making the training set size 1650).

### Environment

- YOLO - YOLOv5s 
- Python: 3.8.0
- OS: Ubuntu 20.04.2 LTS
- PyTorch: torch==1.12.1+cu113

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@freshche the point of the val and train loader tqdm bars is to show you how many images are being loaded. Suggest you view them on train start.
